apps/cowswap-frontend/src/common/pure/OrderProgressBarV2/index.tsx

```diff
@@ -85,6 +107,7 @@ const StepComponent: React.FC<{
 )
 
 const StatusIcon: React.FC<{ status: string; customColor?: string }> = ({ status, customColor }) => {
+  const theme = useTheme()
```

Comment:
Is all you need from the theme the darkMode flag?
Then this should be enough https://github.com/cowprotocol/cowswap/blob/ebb22cec497dfc696d32a2fd75ecc17cfd58eb41/apps/cowswap-frontend/src/legacy/state/user/hooks.tsx#L17

Reply:
Addressed!